scorecard/scorecard/templates/user.html

```diff
@@ -1,48 +1,56 @@
 <html lang="en">
-<head>
+  <head>
     <title>Scorecard: User {{ user }}</title>
+    <link rel="shortcut icon" href="#" />
```

Comment:
Again add the favicon fix. The rest is HTML formatting, not critical if you prefer the style you already had